### PR TITLE
feat: [PIE-6087]: multi service environment for gitops

### PR DESCRIPTION
**Pull request by AshwinBhatkal-Harness**
### Summary

Multi service environment for gitops

#### Screenshots

https://user-images.githubusercontent.com/96711906/200809720-6e501d23-66e7-4aab-b7cb-474470f8f67f.mov

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Review by satyam-harness — APPROVED** [review]

File: src/modules/75-cd/components/PipelineSteps/DeployServiceEntityStep/DeployServiceEntityWidget.tsx [hunk 1]
```diff
@@ -423,8 +423,7 @@ export default function DeployServiceEntityWidget({
           formikRef.current = formik
           const { values } = formik
 
-          // Multi Service is disabled for gitops enabled temporarily
-          const isMultiSvc = !isNil(values.services) && !gitOpsEnabled
+          const isMultiSvc = !isNil(values.services)
```
Comment (bhavya-sinha) [inline src/modules/75-cd/components/PipelineSteps/DeployServiceEntityStep/DeployServiceEntityWidget.tsx:426]: Why is this removed?

**Review by bhavya-sinha — APPROVED** [review]

File: src/modules/75-cd/components/PipelineSteps/DeployServiceEntityStep/DeployServiceEntityWidget.tsx (hunk 1 above)
Comment (AshwinBhatkal-Harness) [reply src/modules/75-cd/components/PipelineSteps/DeployServiceEntityStep/DeployServiceEntityWidget.tsx:426]: We are now supporting multi service with gitops

**Review by kapilkumawat86 — APPROVED** [review]
